app/Models/Lesson.php

```diff
@@ -39,4 +40,24 @@ public function reviews()
     {
         return $this->hasMany(Review::class, 'location_id')->where('locationType', Review::LOCATION_TYPE['lesson']);
     }
+
+    public function getCheckLessonLearnedAttribute()
```

Comment:
tên attribute này khó hiểu quá

Reply:
check xem lesson đã được học bởi user hay chưa ạ

Reply:
nếu thế thì mục đích trả về true/false đúng không nhỉ?
nếu thế tên attribute chưa đúng nhé? nó phải dạng is... chứ